src/Components/Error/Error.tsx

```diff
@@ -4,7 +4,7 @@ import './Error.css'
 
 const Error: FC = () => {
     return (
-        <div className='error'>
+        <div className='error-404'>
```

Comment:
Error styling looks good! Thinking about how we can make this more consistent with colors/styling from app.